### ISSUE TITLE
No basic AUTH working

### ISSUE THREAD

**Issue by Loafter** [deepch/RTSPtoWeb#74]
Hi looks like basic auth for martini not implemented for demo site?
Or it bug only?

**Comment by allenporter** [comment]
The demo site appears to be currently set as public: https://github.com/deepch/RTSPtoWeb/blob/69a751a57886da6cd8bdb649274c5573808c1c11/apiHTTPRouter.go#L46 while the API methods are private.

**Comment by allenporter** [comment]
@deepch I think the options here are:
- lock down the demo server to also use basic auth
- let users turn off demo if they want auth
- do nothing :)

My intuition is that the demo site should also require basic auth for consistency, since it provides a lot of the same functionality as the API, but curious to get your opinion.

**Comment by zardilior** [comment]
I agree with you Allen Porter and might make the change from my side, as I use the GUI to administer things, but can't have RTSP urls exposed everywhere.
